P/PROJ/build_tarballs.jl

```diff
@@ -17,26 +17,13 @@ cd $WORKSPACE/srcdir/proj-*/
 # cross-compiled one since it needs to be executed on the host
 apk add sqlite
 
-if [[ ${target} == *mingw* ]]; then
-    SQLITE3_LIBRARY=${libdir}/libsqlite3-0.dll
-else
-    SQLITE3_LIBRARY=${libdir}/libsqlite3.${dlext}
-fi
```

Comment:
Does this mean that it doesn't need sqlite anymore or it can find it automatically?

Reply:
Still needs it. It just finds the right one with no help now :)

Reply:
> It just finds the right one with no help now :)

[Not sure](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=3426&view=logs&j=a1122a18-1e6d-551f-97f8-cc2b0a7124c6&t=3493a92c-363c-5af1-8f0c-b1c5b6f4653f&l=797) :stuck_out_tongue:

Reply:
Ha what? Pretty sure I tested this locally. Anyway, back to being explicit: 679462cc715cdf7d870ec906848116e5f0b7b6ee

https://proj.org/install.html#cmake-configure-options

Reply:
Hold on, I also need to specify which TIFF to pick up, otherwise it goes wrong:

```
[ Info: Building for x86_64-apple-darwin14-cxx03
[19:24:54] -- Found Sqlite3: /workspace/destdir/lib/libsqlite3.dylib
[19:24:54] -- Sqlite3 version: 3.31.1
[19:24:54] -- Found TIFF: /opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/usr/local/lib/libtiff.dylib (found version "4.0.10") 
[19:24:54] -- Found CURL: /workspace/destdir/lib/libcurl.dylib (found version "7.70.0")
```

Reply:
Ok this should be good to go now!